### PR TITLE
Stupid senseless commit :-1:

### PR DESCRIPTION
**Pull request by eldje**

### CODE REVIEW

**Comment by mastorm** [comment]
ASd

File: Extensions/Core.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public void Setup()
 
         public static string Help()
         {
-            return "Works!";
+            return "Works! Hi from eldje";
```
Comment (mastorm) [inline Extensions/Core.cs]: Great!

File: Extensions/Core.cs [hunk 2]
```diff
@@ -21,7 +21,7 @@ public void Setup()
 
         public static string Help()
         {
-            return "Works!";
```
Comment (eldje) [inline Extensions/Core.cs]: ---

File: Extensions/Core.cs [hunk 3]
```diff
@@ -21,7 +21,7 @@ public void Setup()
```
Comment (eldje) [inline Extensions/Core.cs]: AAAAAAAH! :dancer:

**Pull request merged**